src/components/content/survey-table/survey-table-row/survey-table-row-item/survey-table-row-item.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import '../../survey-table.scss';
+import SurveyTableRowItemText from "./survey-table-row-item-text/survey-table-row-item-text";
+
+
+function SurveyTableRowItem(data) {
+    const {survey} = data;
+    let itemClassName = 'survey-table-item ';
+    itemClassName += survey.opened ? 'selected' : '';
+    return (
+        <div className={itemClassName}>
+            {survey.data.map(d =>
```

Comment:
Why `d`, isn't it `survey`? You should always use readable names, not the shortcuts without any meaning. You could read about best practices [here](https://github.com/ryanmcdermott/clean-code-javascript#avoid-mental-mapping).